src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private final int side1;
+    private final int side2;
```

Comment:
Use meaningful names for variables. 
```suggestion
    private final int width;
    private final int height;
```